README.md

```diff
@@ -143,16 +143,22 @@ ss raw SELECT * FROM npc WHERE form_id > 0xa2c00 AND form_id < 0xa2d00;
 ```
 ## Requirements
 - [SKSE64](https://skse.silverlock.org/)
-- SkyrimSE *1.5.97*
+- SkyrimSE *1.6.323*
 
 ## Build Requirements
 - [Rust](https://www.rust-lang.org/) nightly compiler
 - `rustup target add x86_64-pc-windows-gnu`
+- `rustup install nightly-2020-02-14`
+- The Reason for the heavily outdated Rust version is that the code and libraries contain stuff that's been removed and won't compile. I have never worked with Rust before so I have no Idea how to fix/update the code.
```

Comment:
I am going to make some modifications to master not to depend on outdated Rust version

Reply:
Of course. I have attempted to do so myself but couldn't do it. I did get the compiler to stop complaining about detour by using version 0.8.1 but since I don't know if that would screw something else over I decided to leave it as is, if you can fix it, go right ahead.